internal/provider/provider_test.go

```diff
@@ -223,11 +223,11 @@ func TestAccProvider_clientSecretAuth(t *testing.T) {
 	if os.Getenv("TF_ACC") == "" {
 		t.Skip("TF_ACC not set")
 	}
-	if os.Getenv("ARM_CLIENT_ID") == "" {
-		t.Skip("ARM_CLIENT_ID not set")
+	if os.Getenv("ARM_CLIENT_ID") == "" && os.Getenv("ARM_CLIENT_ID_PATH") == "" {
+		t.Skip("ARM_CLIENT_ID or ARM_CLIENT_ID_PATH not set")
 	}
-	if os.Getenv("ARM_CLIENT_SECRET") == "" {
-		t.Skip("ARM_CLIENT_SECRET not set")
+	if os.Getenv("ARM_CLIENT_SECRET") == "" && os.Getenv("ARM_CLIENT_SECRET_PATH") == "" {
+		t.Skip("ARM_CLIENT_SECRET or ARM_CLIENT_SECRET_PATH not set")
```

Comment:
could we split this out as a separate test?

Reply:
For sure! I'm assuming you mean having one test check `ARM_CLIENT_ID` with `ARM_CLIENT_SECRET` and another to do `ARM_CLIENT_ID_PATH` with `ARM_CLIENT_SECRET_PATH`?

Reply:
yeah exactly 👍 - thanks